src/puppetlabs/puppetdb/cli/shovel.clj

```diff
@@ -0,0 +1,374 @@
+(ns puppetlabs.puppetdb.cli.shovel
```

Comment:
any feelings on changing the name of this ns? I understand why it's called shovel but anyone new to the code is going to find it weird. puppetlabs.puppetdb.cli.queue-migration or something else explicit maybe?